drivers/drivers.go

```diff
@@ -43,6 +43,7 @@ type OSDriver interface {
 	NewSession(path string) OSSession
 	Description() string
 	UriSchemes() []string
+	Publish() string
```

Comment:
Could be called `Flush` as well for a more common concept in I/O

Reply:
I think `Publish()` is more verbose, since it's explicit that it will return a publicly accessible URL.

Reply:
I think I agree with Rafal - `Flush()` has connotations of cleanup, rather than something that returns something useful